packages/kmod-6.1-nvidia/nvidia-persistenced.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=NVIDIA Persistence Daemon
+After=load-tesla-kernel-modules.service load-open-gpu-kernel-modules.service
+# The "nvidia" user must be created before this unit runs.
+Requires=systemd-sysusers.service
+After=systemd-sysusers.service
```

Comment:
Since you're using the default unit dependencies (i.e. you don't have `DefaultDependencies=no`, which is correct) then you can omit these - they're guaranteed to run before `sysinit.target` is reached.

```suggestion
```